src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -366,6 +376,26 @@ class ContentSidebar extends PureComponent<Props, State> {
         this.errorCallback(e);
     };
 
+    /**
+     * Handles a failed file user info fetch
+     *
+     * @private
+     * @param {Error} e - API error
+     * @return {void}
+     */
+    fetchCurrentUserErrorCallback = (e: Error) => {
+        this.setState({
+            currentUser: undefined,
+            currentUserError: {
```

Comment:
What is your plan for using this?

Reply:
Not sure yet. In the case that it returns, it means that an auth token was made for/by no one, which I'm not sure how possible that is. Anyone else know?

Reply:
Talking to Zach about this, right now